docker/nginx/nginx.conf

```diff
@@ -0,0 +1,88 @@
+#ssl_certificate /etc/letsencrypt/live/myopenhab.org/fullchain.pem;
+#ssl_certificate_key /etc/letsencrypt/live/myopenhab.org/privkey.pem;
```

Comment:
My view is this, out of the box we should aim for users to have a secure setup. I download it, install it and I have everything working securely.  If users have their own certs or need anything other then our simple setup they will need to customize the docker file in any case, no?